### DISCUSSION TITLE
Error Compiling and Testing MOOSE

### DISCUSSION THREAD

**Discussion by ibuhari86** [idaholab/moose#21479] [Q&A Getting Started] [answered]
Hello Everyone,

I am new to the group and to MOOSE. I tried to follow the instructions on the website to install MOOSE for the first time, however, I run into this problem (attached). Any suggestions on how to resolve this issue? Thank you.

Regards,
BI
![Screenshot 2022-07-01 090947](https://user-images.githubusercontent.com/68480824/176905040-ab1afe43-47eb-4aa3-b609-21cee9b290b0.png)
.

**Comment by GiudGiud** [comment, marked as answer]
Hello

It looks like it's not finding libmesh. Did you `mamba install moose-libmesh` ?
Were you following these instructions?
https://mooseframework.inl.gov/getting_started/installation/conda.html

Guillaume

**Reply by ibuhari86** [reply]
Hi Guillaume, thanks for your response. Yes, those are the instructions I followed and everything seemed to be going alright up until when I wanted to compile and test. I actually just redid it to make sure I did not miss any step, and it still gave me the same error message. Thank you
![Screenshot 2022-07-01 102422](https://user-images.githubusercontent.com/68480824/176913410-b7635a73-b082-48f9-b2c4-e4f4383c1cd9.png)
.

**Reply by GiudGiud** [reply]
this time you forgot to activate the moose environment. You re still in (base)

**Reply by ibuhari86** [reply]
I did that, same issue. Not really sure what is going on.
![Screenshot 2022-07-01 121639](https://user-images.githubusercontent.com/68480824/176932810-14553eac-ac2f-45e8-ac4d-1a8d6380f41a.png)

**Reply by GiudGiud** [reply]
what is the output of `mamba list `?

**Reply by ibuhari86** [reply]
This is what I get with that command.
![Screenshot 2022-07-01 154320](https://user-images.githubusercontent.com/68480824/176959683-c777f76a-5b3e-4557-b5f0-303f43e8521b.png)

**Reply by GiudGiud** [reply]
either your mamba is broken and you need to re-instsall it or you did not install the moose dependencies like in the instructions. 
Did you run `mamba install moose-libmesh` ?

**Reply by ibuhari86** [reply]
Hello Guillaume, 

Thank you for your patience. I just don't know what it is, but I removed everything and reinstalled it again but still run into the same errors. And yes, I did mamba install moose-libmesh. Please see attached images. Thank you.
![Screenshot 2022-07-02 142801](https://user-images.githubusercontent.com/68480824/177012554-bf277a32-1343-4175-87f7-f62b6b489737.png)
![Screenshot 2022-07-02 143250](https://user-images.githubusercontent.com/68480824/177012557-7944f9a3-f0ee-4e0f-a4c7-1d2c71d9463f.png)

**Reply by hugary1995** [reply]
Can you show us the output of `cat ~/.condarc`

Very likely there is a deprecated channel.

**Reply by GiudGiud** [reply]
good call. I see idaholab in the list of channels it's pulling from.
Once you have pasted the content of that file, please delete that file then try again. You will need to run:
conda config --add channels https://conda.software.inl.gov/public

**Reply by ibuhari86** [reply]
This is it.
![Screenshot 2022-07-02 162908](https://user-images.githubusercontent.com/68480824/177015345-52b7705c-e23a-42cc-be2f-6fd63afe5939.png)

**Reply by GiudGiud** [reply]
the idaholab channel could be the trouble maker here, we no longer use it

**Reply by ibuhari86** [reply]
How do I delete it? Since this does not seem to be located in a folder.

**Reply by ibuhari86** [reply]
Or could I just try reinstall it and then use "conda config --add channels https://conda.software.inl.gov/public" instead of the idaholab's?

**Reply by GiudGiud** [reply]
that file can be modified or deleted

**Reply by ibuhari86** [reply]
I now have the correct channels, with the 'idaholab' deleted, error still persists.
![Screenshot 2022-07-02 172252](https://user-images.githubusercontent.com/68480824/177016636-cf63d866-df5b-442c-b0e0-c3d4f55b0a39.png)

**Reply by GiudGiud** [reply]
this /usr/bin/env python : no such file or directory is problematic.

At this point, if `mamba list` does like the packages we ask you to download (what does it return??), then it's a python conflict error.

**Reply by hugary1995** [reply]
After fixing the channels, you have to recreate the mamba moose environment, i.e. reinstall the packages.

**Reply by ibuhari86** [reply]
Hello, 

Finally, it worked after I recreated the mamba environment. Thank you Giullaume and hugary1995 for your inputs and patience. 

Regards,
BI.
![Screenshot 2022-07-02 195049](https://user-images.githubusercontent.com/68480824/177019326-aa1c5002-6f3e-4c87-a98e-b11e5249a572.png)

**Reply by Oops-Qiao** [reply]
Hi brother :)

I have the same problem these two days. I wonder if I can reach you to ask for a little help for it. Thank you.

**Reply by ibuhari86** [reply]
Hi Oops-Qiao, 

I am also very new to MOOSE, but I might be able to help if it is similar to the problem I had. Thank you.

**Reply by milljm** [reply]
> After fixing the channels, you have to recreate the mamba moose environment, i.e. reinstall the packages.

What @hugary1995 said, is going to be the answer after fixing (removing) the old `idaholab` channel.
